docs/TabbedShowLayout.md

```diff
@@ -345,9 +345,111 @@ const StaticPostShow = () => (
 
 When passed a `record`, `<TabbedShowLayout>` creates a `RecordContext` with the given record.
 
+## Role-Based Access Control (RBAC)
+
+Fine-grained permissions control can be added by using the `<TabbedShowLayout>` component provided by the `@react-admin/ra-rbac` package.
+
+Use it in conjuction with [`<TabbedShowLayout.Tab>`](#tabbedshowlayouttab) and add a `name` prop to the `Tab` to define the resource on which the user needs to have the 'read' permissions for.
```

Comment:
```suggestion
Use it in conjunction with ra-rbac's [`<TabbedShowLayout.Tab>`](#tabbedshowlayouttab-1) and add a `name` prop to the `Tab` to define the resource on which the user needs to have the 'read' permissions for.
```

same

Reply:
So this typo is definitely in EE documentation too